integration/integration_test.go

```diff
@@ -82,18 +82,18 @@ func TestHttpIngress(t *testing.T) {
 					assert.True(t, ok, "good_stuff is not a string")
 					assert.Equal(t, "This is good stuff", goodStuff)
 				}),
-				httpCall(rd, http.MethodPost, "/users", jsonData(t, obj{"userID": 123, "postID": 345}), func(t testing.TB, resp *httpResponse) {
```

Comment:
How come these didn't fail the casing change release?

Reply:
it was bad timing - the integration tests started silently failing to execute sometime between yesterday and today because `strcase.ToCamel` seems to have been removed (we had to start using `strcase.ToUpperCamel` per [this change](https://github.com/TBD54566975/ftl/pull/902)).

here's the CI run from your casing change: https://github.com/TBD54566975/ftl/actions/runs/7827634224/job/21355851494

the test extraction step was failing with 
```
integration/integration_test.go:164:47: undefined: strcase.ToCamel
integration/integration_test.go:273:23: undefined: strcase.ToCamel
```
so the run flag was provided with empty input

Reply:
Ah okay. Is the extraction step fixed so this doesn't happen again?

Also BTW I'd suggest using `grep '^func Test.*'` to extract the test names, mainly because it's _much_ faster than using `go test -list`, which has to compile the code first before it's able to continue.